### PR TITLE
update buildtools to a bitrise supported one + update MAS

### PR DESCRIPTION
**Pull request by tobrun**
The CI bots on this repo were having the same issues as the one on gl-native in mapbox/mapbox-gl-native#7729. This PR upgrades the build tools to a bitrise supported version.

Review @zugaldia

### CODE REVIEW

**Comment by tobrun** [comment]
it seems that master isn't able to resolve a drawable resource, 
picking this up with this PR to make CI succeed.

File: build.gradle [hunk 1]
```diff
@@ -1,8 +1,9 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
   repositories {
-    mavenCentral()
     jcenter()
+    maven { url "https://jitpack.io" }
```
Comment (zugaldia) [inline build.gradle]: @tobrun What are we using jitpack for?

Comment (tobrun) [inline build.gradle]: https://github.com/javiersantos/MaterialStyledDialogs, but this change shouldn't have been in here as it's already included in `app/build.gradle`. Should have waited for a rereview.

Comment (tobrun) [inline build.gradle]: removal picked up here https://github.com/mapbox/mapbox-android-demo/pull/196/files

**Pull request merged**